likecoin/admin/metabox.php

```diff
@@ -222,63 +267,77 @@ function likecoin_add_publish_meta_box( $publish_params ) {
 			<span class="dashicons dashicons-image-rotate" style="vertical-align:middle"></span>
 		</button>
 	</h3>
+	<div id="lcTitleStatus"><?php if ( ! empty( $iscn_status['iscn_card_url'] ) ) { ?>
+		<h1 class="iscn-status-green"> &#183; </h1><h3 class="iscn-status-text"><?php esc_html_e( 'LIVE on #DePub', LC_PLUGIN_SLUG ); ?></h3>
+		<?php } elseif ( 'publish' === $wordpress_publish_status ) { ?>
+			<h1 class="iscn-status-orange"> &#183; </h1><h3 class="iscn-status-text"><?php esc_html_e( 'READY to #DePub', LC_PLUGIN_SLUG ); ?></h3>
+		<?php } else { ?>
+			<h1 class="iscn-status-red"> &#183; </h1><h3 class="iscn-status-text"> <?php esc_html_e( 'Publish Your Post First', LC_PLUGIN_SLUG ); ?></h3>
+		<?php } ?>
+	</div>
 	<table class="form-table">
 		<tbody>
+			<tr id="likecoin_submit_arweave">
+				<th><label><?php esc_html_e( 'ISCN Status', LC_PLUGIN_SLUG ); ?></label></th>
+				<td id="lcISCNStatus">
+					<?php if ( ! empty( $iscn_status['iscn_card_url'] ) ) { ?>
+						<a rel="noopener" target="_blank" href="<?php echo esc_url( $iscn_status['iscn_card_url'] ); ?>">
+							<?php echo esc_html( $iscn_status['iscn_id'] ); ?>
+						</a>
+					<?php } elseif ( 'publish' === $wordpress_publish_status ) { ?>
+												<span id="lcArweaveUpload">
+						<button id="lcArweaveISCNBtn" class="button button-primary">
+							<?php esc_html_e( 'Submit to ISCN', LC_PLUGIN_SLUG ); ?>
+						</button>
+					</span>
+					<?php } else { ?>
+						<button id="lcArweaveISCNBtn" class="button button-primary" disabled>
+							<?php esc_html_e( 'Submit to ISCN', LC_PLUGIN_SLUG ); ?>
+						</button>
+					<?php } ?>
```

Comment:
Should put into same if block, and use `'publish' === $wordpress_publish_status` to determine if `disabled` should be added

Reply:
Where is `id=lcArweaveUpload` used?

Reply:
lcArweaveUpload is not used, will delete the span

Reply:
will put into the same block